datajoint_utilities/dev_helpers.py

```diff
@@ -0,0 +1,56 @@
+import datajoint as dj
+from pymysql import OperationalError
+
+"""
+Dev helper functions. Chris Brozdowski <CBroz@datajoint.com>
+- list_schemas_prefix: returns a list of schemas with a specific prefix
+- drop_schemas: Cycles through schemas on a given prefix until all are dropped
+"""
+
+
+def list_schemas_prefix(prefix):
+    """Returns list of schemas with a specific prefix"""
+    return [s for s in dj.list_schemas() if s.startswith(prefix)]
+
+
+def drop_schemas(prefix=None, dry_run=True, force_drop=False):
```

Comment:
```suggestion
def drop_schemas(prefix, dry_run=True, force_drop=False):
```
I'd suggest to make `prefix` not optional, users should explicitly provide this, being that they retrieve from dj.config themselves or elsewhere